frontend/lib/norent/letter-builder/national-metadata.tsx

```diff
@@ -10,17 +10,20 @@ import {
 import { LosAngelesZipCodes } from "../data/la-zipcodes";
 
 type StateLawForBuilderEntry = {
-  linkToLegislation: string;
+  linkToLegislation?: string;
   textOfLegislation: string;
+  stateWithoutProtections: boolean;
 };
 
+export enum CovidStateLawVersion {
+  V1_NON_PAYMENT = "V1 non-payment",
+  V2_HARDSHIP = "V2 hardship",
+  V3_FEW_PROTECTIONS = "V3 few protections",
+}
+
 type StateLawForLetterEntry = {
-  textOfLegislation1: string;
-  textOfLegislation2?: string;
-  textOfLegislation3?: string;
-  textOfLegislation4?: string;
-  textOfLegislation5?: string;
-  textOfLegislation6?: string;
+  whichVersion: CovidStateLawVersion;
```

Comment:
@sraby the "version" of the state law for letter is a TypeScript [Enum](https://www.typescriptlang.org/docs/handbook/enums.html), not sure if you've used those before but see the test case code in this PR for an example of how they're used!

Reply:
Oh cool! I def remember working with them before but will take a look at your test case code for a refresher. Thanks!